packages/alfa-scraper/src/screenshot.ts

```diff
@@ -0,0 +1,136 @@
+import { Equatable } from "@siteimprove/alfa-equatable";
+import { Serializable } from "@siteimprove/alfa-json";
+
+import * as json from "@siteimprove/alfa-json";
+
+export class Screenshot implements Equatable, Serializable {
```

Comment:
I do agree that the name can be a little deceptive. We _could_ introduce an additional package that defines a simple virtual file format, with things like `File.Type`, `File<T extends File.Type>`, and `File.Meta<T extends File.Type>`, but I do feel it might be a little premature.

Reply:
I'll be keeping the name for now for the simple reason that it aligns with the `--screenshot` flag. Let's revisit the virtual file format if the need arises.